networkx/algorithms/isomorphism/isomorph.py

```diff
@@ -24,6 +24,10 @@ def could_be_isomorphic(G1, G2):
     Notes
     -----
     Checks for matching degree, triangle, and number of cliques sequences.
+    The triangle sequence contains the number of triangles each node is part of.
+    The clique sequence contains for each node the size of the maximal clique
+    involving that node.
```

Comment:
```suggestion
    The clique sequence contains for each node the number of maximal cliques
    involving that node.
```
Usually, a node will be part of more than one maximal clique. A maximal clique cannot add any nodes while remaining a clique. That is different from the maximum clique for a node which is the largest of all the maximal cliques.  This function checks the number of maximal cliques in which this node participates.